mne/time_frequency/tfr.py

```diff
@@ -1107,9 +1162,246 @@ def plot(self, picks, baseline=None, mode='mean', tmin=None, tmax=None,
                         ylim=None, tfr=data[idx: idx + 1], freq=freqs,
                         x_label='Time (ms)', y_label='Frequency (Hz)',
                         colorbar=colorbar, cmap=cmap, yscale=yscale)
+
+            if title is 'auto':
+                if aggregate is None or len(info['ch_names']) == 1:
+                    title = info['ch_names'][0]
+                else:
+                    ch_title = ", ".join(info['ch_names'][:6])
+                    if len(info['ch_names']) > 6:
+                        ch_title += ", ..."
+                    title = 'Aggregate (%s) of %s' % (aggregate, ch_title)
+
             if title:
                 fig.suptitle(title)
 
+            plt_show(show)
+            return fig, data, times*1e-3, freqs
+
+
+    @verbose
+    def plot_joint(self, picks=None, exclude=None, baseline=None, mode='mean',
+             tmin=None, tmax=None, fmin=None, fmax=None, vmin=None, vmax=None,
+             cmap='RdBu_r', dB=False, colorbar=True, show=True, title=None,
+             layout=None, yscale='auto', verbose=None, aggregate='mean',
+             timefreqs=None, timefreqs_average=None, topomap_args=None):
+        """Plot TFRs as a two-dimensional image(s) with topomaps at
+        specific times and frequencies.
```

Comment:
function / methods / classes should have single-line first line, so you can just stop after "topomaps here". If you want to elaborate, add a new paragraph below.